src/pages/das-api/methods/get-nft-editions.md

```
@@ -0,0 +1,34 @@
+---
+title: Get NFT Editions
+metaTitle: Get NFT Editions | DAS API
+description: Get all printable editions of a master edition NFT mint
+---
+
+Returns all printable editions for a master edition NFT mint, including edition numbers, addresses, and supply information. You can also pass in an edition address and will receive the master edition plus the other editions that are part of the same master edition.
+## Parameters
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)